marklogic-data-hub-central/ui/src/pages/Run.tsx

```diff
@@ -34,7 +34,7 @@ const Run: React.FC = () => {
     const authorityService = useContext(AuthoritiesContext);
     const canReadFlow = authorityService.canReadStep();
     const canWriteFlow = authorityService.canWriteStep();
-    const hasOperatorRole = authorityService.hasOperatorRole();
+    const hasOperatorRole = authorityService.canRunStep();
```

Comment:
Currently we have tests for Run tile in Run.test.tsx, but they are run as developer and operator. Should we run them as flow-writer and step runner instead, so we would have verified a flow can be run with a lesser and more appropriate authority?

Reply:
wouldn't hurt. I'll adjust that.